src/Uno.Extensions.Reactive/Uno.Extensions.Reactive.csproj

```diff
@@ -8,6 +8,8 @@
 			As we are InternalsVisibleTo(Uno.Extensions.Reactive.Tests), we disable some compatibility types that are not used by Reactive package itself.
 		-->
 		<UnoExtensionsGeneration_DisableModuleInitializerAttribute>True</UnoExtensionsGeneration_DisableModuleInitializerAttribute>
+		<!-- TODO: Address nullability warnings -->
```

Comment:
@dr1rrb will probably need your assistance to address this TODO because there's a lot of nullability warnings as a result of including net6 tfm

Reply:
This can be disregarded as I've backed out most of these changes :-)